pkg/cmd/root/root.go

```diff
@@ -70,27 +73,15 @@ func NewCobraCmd(rootCmd *RootCmd, f *cmdutil.Factory) *cobra.Command {
 		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
 			startTime = time.Now()
 			logger.LogLevel(f.Logger)
-			err := doPreCommandCheck(cmd, f, PreCmd{
+
+			if strings.HasPrefix(configFlag, PREFIX_FLAG) {
+				return fmt.Errorf("A configuration path is expected for your location, not a flag")
+			}
+
+			return doPreCommandCheck(cmd, f, PreCmd{
 				config: configFlag,
 				token:  tokenFlag,
 			})
-			if err != nil {
-				return err
-			}
-			return nil
-		},
-		PersistentPostRunE: func(cmd *cobra.Command, args []string) error {
-			executionTime := time.Since(startTime).Seconds()
-
-			//1 = authorize; anything different than 1 means that the user did not authorize metrics collection, or did not answer the question yet
-			if globalSettings.AuthorizeMetricsCollection != 1 {
```

Comment:
This has to be somewhere 
If user did not allow for metrics collection, we just skip the process